### ISSUE TITLE
Reshape is not converting in some cases.

### ISSUE THREAD

**Issue by andrew-yang0722** [onnx/onnx-tensorflow#756]
Converting is not working in onnx::Reshape() which is converted from .view() of pytorch.
I got 0 dim output by When infer the tflite modelTF Lite on android.
That is tflite.getOutputTensor().numDimensions() is 0.

I think converting failed.

I make a sample codes that shows this problem.

Pytorch codes
```python
class ReshapeModel(nn.Module):  # 00 안 됨
    def __init__(self):
        super(ReshapeModel, self).__init__()

    def forward(self, x):
        out = x.view(1, -1, 4).contiguous() # 안 됨.
        return out

dummy_input =  torch.randn(1, 56, 56, 2)
```

Onnx log
``` log
graph(%input : Float(1, 56, 56, 2)):
  %1 : Tensor = onnx::Constant[value= 1 -1  4 [ CPULongType{3} ]]()
  %output : Float(1, 1568, 4) = onnx::Reshape(%input, %1) # simple_onnx.py:28:0
  return (%output)
```

In some cases, reshape is working, and some cases it fails.
Fail case: torch.randn(1, 56, 56, 2) -> x.view(1, 56, -1, 4)
Fail case: torch.randn(1, 56, 56, 2) -> x.view(1, -1, 4) 
OK case: torch.randn(1, 56, 2) -> x.view(1, -1)
OK case: torch.randn(1, 56, 56, 2) -> x.view(1, -1)

tensorflow             1.15.0
onnx                   1.7.0
onnx-tf                1.6.0   tf-1.x branch
torch                  1.4.0

**Comment by chinhuang007** [comment]
Can you provide the onnx file that is generated from pytorch code?

**Comment by andrew-yang0722** [comment]
Here is the file.
[onnx_file](https://www.dropbox.com/s/2wgsau315m0lejg/simple_reshape_00.onnx?dl=0)
